docs/tutorial/learning-resources.rst

```diff
@@ -1,20 +1,22 @@
 .. meta::
-   :description: Omniperf external training resources
-   :keywords: Omniperf, ROCm, profiler, tool, Instinct, accelerator, AMD,
-              training, examples
+   :description: ROCm Compute Profiler external training resources
+   :keywords: Omniperf, examples, tutorials, videos, lesson, lessons, how
 
 ******************
 Learning resources
 ******************
 
-This section is a catalog of external resources and third-party content that
-can help you learn Omniperf. Some areas of the following content might be
-outdated.
+This section provides a curated list of external resources and third-party
+content to support learning the ROCm Compute Profiler. Some information in
+these materials may be outdated.
 
-Introduction to Omniperf
+ROCm Compute Profiler was previously known as Omniperf. Some of the following
+resources use the earlier name.
+
+Introduction to ROCm Compute Profiler
   :fab:`youtube` `AMD profiling workshop (Pawsey Supercomputing Research Centre) <https://www.youtube.com/watch?v=9AkxBCiInCw>`_
 
-Omniperf example exercises
+ROCm Compute Profiler example exercises
   `<https://github.com/amd/HPCTrainingExamples/tree/main/OmniperfExamples>`__
```

Comment:
Will these examples be updated w/ the rename? https://github.com/amd/HPCTrainingExamples/tree/main/Omniperf-OpenMP/Fortran/1_collapse